.github/CODEOWNERS

```diff
@@ -83,33 +83,33 @@ Makefile                                                 @getsentry/owners-sentr
 /tests/tools/mypy_helpers                                @getsentry/python-typing
 
 ## GitHub Routing Automations - notion.so/473791bae5bf43399d46093050b77bf0
-/.github/labels.yml                                      @getsentry/open-source
-/.github/workflows/react-to-product-owners-yml-changes.yml @getsentry/open-source
-/bin/react-to-product-owners-yml-changes.py              @getsentry/open-source
-/bin/react-to-product-owners-yml-changes.sh              @getsentry/open-source
-/static/app/components/sidebar/index.tsx                 @getsentry/open-source
+/.github/labels.yml                                      @getsentry/dev-infra
+/.github/workflows/react-to-product-owners-yml-changes.yml @getsentry/dev-infra
+/bin/react-to-product-owners-yml-changes.py              @getsentry/dev-infra
+/bin/react-to-product-owners-yml-changes.sh              @getsentry/dev-infra
+/static/app/components/sidebar/index.tsx                 @getsentry/dev-infra
 
 ## Backup - getsentry/team-ospo#153
```

Comment:
Same comment here for backup/relocation. Last I heard some folks on hybrid cloud might be the owners here. @bc-sentry

Reply:
Can we merge as-is and then if you want to hand off from dev-infra to another team do that separately?

Reply:
You can assign relocation to @getsentry/hybrid-cloud as there are a few folks with context on it.

Reply:
Done in 15c1c3f458094324c4c568ed112308b511b836d8, thank you!